src/components/taskbar/index.js

```diff
@@ -51,7 +51,7 @@ const TaskBar = ({ classes, goHome, goRefund }) => (
         alt="logo"
       />
       <span className={classes.logoText}>Boltz</span>
-      <span className={classes.subLogoText}>ALPHA</span>
+      <span className={classes.subLogoText}>ALPHA (TESTNET ONLY)</span>
```

Comment:
What about writing `TESTNET ALPHA`? @peartobear

Reply:
Yeah. That sounds good to me as well.